da/celestia/mock/server.go

```diff
@@ -89,10 +89,14 @@ func (s *Server) submit(w http.ResponseWriter, r *http.Request) {
 	}
 
 	res := s.mock.SubmitBlock(&block)
+	code := 0
+	if res.Code != da.StatusSuccess {
+		code = 3
```

Comment:
why not simply pass the res.Code here?

Reply:
Because it's code from different API, and thus values differ. `da.StatusSuccess` is `1`, and `TxResponse.Code` use `0` to denote success.

Reply:
i see. unfortunately, 0 != 1 😄